DownFall/Models/AttackModel.swift

```diff
@@ -8,78 +8,119 @@
 
 import Foundation
 
+struct AttackSlope: Equatable, Decodable {
+    let over: Int
+    let up: Int
+    
+    static var south: AttackSlope {
+        return AttackSlope(over: 0, up: -1)
+    }
+    
+    static var sideways: [AttackSlope] {
+        return [AttackSlope(over: -1, up: 0), AttackSlope(over: 1, up: 0)]
+    }
+    
+    static var diagonals: [AttackSlope] {
+        return [AttackSlope(over: -1, up: 1),
+                AttackSlope(over: 1, up: 1),
+                AttackSlope(over: -1, up: -1),
+                AttackSlope(over: 1, up: -1)]
+    }
+    
+    static var playerPossibleAttacks: [AttackSlope] {
+        return [
+            AttackSlope(over: -1, up: 0),
+            AttackSlope(over: 0, up: 1),
+            AttackSlope(over: 0, up: -1),
+            AttackSlope(over: 1, up: 0)
+        ]
+    }
```

Comment:
A lot of theses were created only for tests.  Move those to a file that only targets the tests